vhdbuilder/packer/test/run-test.sh

```diff
@@ -10,108 +11,137 @@ RESOURCE_GROUP_NAME="$TEST_RESOURCE_PREFIX-$(date +%s)"
+  else
+    az group delete --name $RESOURCE_GROUP_NAME --yes --no-wait
+  fi
 }
 trap cleanup EXIT
 
 DISK_NAME="${TEST_RESOURCE_PREFIX}-disk"
 VM_NAME="${TEST_RESOURCE_PREFIX}-vm"
 
 if [ "$MODE" == "sigMode" ]; then
-	echo "SIG existence checking for $MODE"
-	id=$(az sig show --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME}) || id=""
-	if [ -z "$id" ]; then
-		echo "Shared Image gallery ${SIG_GALLERY_NAME} does not exist in the resource group ${AZURE_RESOURCE_GROUP_NAME} location ${AZURE_LOCATION}"
-        exit 1
-	fi
+  echo "SIG existence checking for $MODE"
+  id=$(az sig show --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME}) || id=""
+  if [ -z "$id" ]; then
+    echo "Shared Image gallery ${SIG_GALLERY_NAME} does not exist in the resource group ${AZURE_RESOURCE_GROUP_NAME} location ${AZURE_LOCATION}"
+    exit 1
+  fi
 
-	id=$(az sig image-definition show \
-		--resource-group ${AZURE_RESOURCE_GROUP_NAME} \
-		--gallery-name ${SIG_GALLERY_NAME} \
-		--gallery-image-definition ${SIG_IMAGE_NAME}) || id=""
-	if [ -z "$id" ]; then
-		echo "Image definition ${SIG_IMAGE_NAME} does not exist in gallery ${SIG_GALLERY_NAME} resource group ${AZURE_RESOURCE_GROUP_NAME}"
-        exit 1
-	fi
+  id=$(az sig image-definition show \
+    --resource-group ${AZURE_RESOURCE_GROUP_NAME} \
+    --gallery-name ${SIG_GALLERY_NAME} \
+    --gallery-image-definition ${SIG_IMAGE_NAME}) || id=""
+  if [ -z "$id" ]; then
+    echo "Image definition ${SIG_IMAGE_NAME} does not exist in gallery ${SIG_GALLERY_NAME} resource group ${AZURE_RESOURCE_GROUP_NAME}"
+    exit 1
+  fi
 
-    IMG_DEF="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/galleries/${SIG_GALLERY_NAME}/images/${SIG_IMAGE_NAME}/versions/${SIG_IMAGE_VERSION}"
+  IMG_DEF="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/galleries/${SIG_GALLERY_NAME}/images/${SIG_IMAGE_NAME}/versions/${SIG_IMAGE_VERSION}"
 
-    # In SIG mode, Windows VM requires admin-username and admin-password to be set,
-    # otherwise 'root' is used by default but not allowed by the Windows Image. See the error image below:
-    # ERROR: This user name 'root' meets the general requirements, but is specifically disallowed for this image. Please try a different value.
-    az vm create\
+  # In SIG mode, Windows VM requires admin-username and admin-password to be set,
+  # otherwise 'root' is used by default but not allowed by the Windows Image. See the error image below:
+  # ERROR: This user name 'root' meets the general requirements, but is specifically disallowed for this image. Please try a different value.
+  az vm create \
     --resource-group $RESOURCE_GROUP_NAME \
     --name $VM_NAME \
     --image $IMG_DEF \
     --admin-username $Test_VM_ADMIN_USERNAME \
     --admin-password $TEST_VM_ADMIN_PASSWORD \
```

Comment:
--admin-username $Test_VM_ADMIN_USERNAME \
--admin-password $TEST_VM_ADMIN_PASSWORD \

Why the capitalization difference: one is $Test_***, the other is $TEST_***. (I understand this is not new code. Just curious why.)

Reply:
:), yeah that doesn't seem right, fixing it :)